client/components/BikeItem.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { View, Text, StyleSheet, Image, TouchableOpacity } from 'react-native';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import { Bike } from '../types/types';
+
+const BikeItem = (props: Bike) => {
```

Comment:
I thought that maybe we would share a component for bike data display but I think it makes sense to have them separate like this.